cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,33 @@
+import time
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        db_conn = None
+        retry_delay = 1
+        max_retries = 5
+        retries = 0
+
+        while not db_conn and retries < max_retries:
+            try:
+                db_conn = connections["default"]
+            except OperationalError:
+                retries += 1
+                self.stdout.write(f"Database unavailable, retrying in"
+                                  f" {retry_delay} seconds..."
+                                  f" (attempt {retries}/{max_retries})")
+                time.sleep(retry_delay)
```

Comment:
There is a logical error in your code. You are checking the database connection by simply accessing the `connections['default']` dictionary. This will not raise an OperationalError if the database is not available. Instead, it will return a connection object that may not be open. You should use the `ensure_connection` method of the connection object to actually test the connection. Here is a suggestion for how you could modify your code:

try:
    db_conn = connections['default']
    db_conn.ensure_connection()
except OperationalError:
    ...
```
This will ensure that an OperationalError is raised if the database is not available, and your retry logic will work as expected.